server/svix-server/src/core/idempotency.rs

```diff
@@ -445,7 +445,7 @@ mod tests {
         // Generate a new token so that keys are unique
```

Comment:
This PR needs a better description.
https://github.com/svix/svix-webhooks/pull/488#issuecomment-1148902722